listener.py

```diff
@@ -0,0 +1,39 @@
+import socket
```

Comment:
Change indentation to use 4 spaces instead of tabs, this is not consistent with the file connection.py

Reply:
Fixed :) https://github.com/roy-cohen273/cardazim/commit/4bbe2c214b2842ff006258b6144ceb8060e188af